kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -1000,6 +1008,117 @@ def check_ceph_fencer(self):
+
+        return True
+
+    def read_fencer_heartbeat(self, host_uuid, ps_uuid):
+        # type: (str, str) -> (float, list[str])
+        status = self._read_heartbeat_file()
+        return status.heartbeat_time, status.vm_uuids
+
+    def exec_fencer(self):
+        try:
+            self._exec_fencer()
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)